plugins/linear-genome-view/src/LinearGenomeView/components/Highlight.tsx

```diff
@@ -140,4 +144,20 @@ const Highlight = observer(function Highlight({ model }: { model: LGV }) {
   ) : null
 })
 
-export default Highlight
+const HighlightGroup = observer(function HighlightGroup({
+  model,
+}: {
+  model: LGV
+}) {
+  if (!model.highlight) {
+    return null
+  }
+
+  return model.highlight
+    ? model.highlight.map((h, idx) => {
+        return <Highlight key={idx} model={model} highlight={h} />
```

Comment:
if possible a more precise key would be good. can json stringify "h" for example